docs/system/yocto/yocto-build-manually.md

```diff
@@ -45,13 +45,20 @@ repo sync --force-sync
 
 Set up environment variables:
 ```bash
-export DIR_WORK=$PWD/../
+export DIR_WORK=$PWD
 export BUILD_TAG=*optional build Tag*
 export PLATFORM_VERSION="$(git -C $DIR_WORK.repo/manifests rev-parse --short HEAD)"
 export PLATFORM_VERSION_DETAILS="$(repo forall -c 'echo $REPO_PATH\nLREV: $REPO_LREV\nRREV: $REPO_RREV; git diff --stat -b $REPO_LREV..HEAD ; echo -n "Commit: " ; git rev-parse HEAD ; echo -n "Uncommited changes: " ; git status -b -s ; git diff --stat -b ; echo ')"
 echo "building with repo versions: $PLATFORM_VERSION"
 export BB_ENV_PASSTHROUGH_ADDITIONS="$BB_ENV_PASSTHROUGH_ADDITIONS BUILD_TAG PLATFORM_VERSION PLATFORM_VERSION_DETAILS"
-export TEMPLATECONF=$PWD/../sources/meta-mobility-poky-distro/conf
+#old before kirkstone export TEMPLATECONF=$PWD/sources/meta-mobility-poky-distro/conf
```

Comment:
This comment is confusing to me.

Reply:
```suggestion
#used to be TEMPLATECONF=$PWD/sources/meta-mobility-poky-distro/conf (before kirkstone export)
```
P.S: I'm unsure why this comment is useful, and what kirkstone export is. Is this here for internal reasons?

Reply:
In the distro we changed where the bblayer and conf is located and we did it in partlly in kirkstone but if we want tu build older release tags this will be the one to change. It might be good to remove later when we have left kirkstone.